gno.land/cmd/gnoland/start.go

```diff
@@ -241,12 +241,18 @@ func execStart(c *startCfg, io commands.IO) error {
 	// Initialize the log format
 	logFormat := log.Format(strings.ToLower(c.logFormat))
 
+	if logFormat != log.JSONFormat {
+		io.Println(startGraphic)
+	}
+
 	// Initialize the zap logger
 	zapLogger := log.GetZapLoggerFn(logFormat)(io.Out(), logLevel)
 
 	// Wrap the zap logger
 	logger := log.ZapLoggerToSlog(zapLogger)
 
+	slog.SetDefault(logger)
```

Comment:
Not sure about this. I don't care if we use the `slog` default logger, but we should use it consistently throughout or not use it at all. I don't want to mix `slog.Default` and non-default `logger`.